src/discrete-layer/components/map-container/map-container.tsx

```diff
@@ -130,32 +138,89 @@ export const MapContainer: React.FC<MapContainerProps> = (props) => {
 
   return (
     <div className="map">
-      <div className="filtersPosition" style={{backgroundColor: theme.primary, width: props.mapActionsWidth}}>
+      <div className="filtersPosition" style={{backgroundColor: theme.primary, width: mapActionsWidth}}>
         <div className="filtersContainer">
           <PolygonSelectionUi
             onCancelDraw={onCancelDraw}
             onReset={onReset}
             onStartDraw={setDrawType}
             isSelectionEnabled={isDrawing}
             onPolygonUpdate={onPolygonSelection}
-            mapActionsWidth = {props.mapActionsWidth}
-            handleOtherDrawers={props.handleOtherDrawers}
+            mapActionsWidth = {mapActionsWidth}
+            handleOtherDrawers={(): void => setFiltersOpen(false)}
           />
-          {props.filters?.map((filter, index) => (
-            <div key={index} className="filtersMargin">
-              {filter}
-            </div>
-          ))}
+          <div className="filtersMargin">
+            <Button
+              outlined
+              theme={['primaryBg', 'onPrimary']}
+              onClick={(): void => setFiltersOpen(!filtersOpen)}
+              icon="filter_alt"
+            >
+              <FormattedMessage id="filters.title" />
+            </Button>
+            {filtersOpen && (
+              <Box className="drawerPosition" style={{  height: '300px', width: mapActionsWidth}}>
+                <Drawer dismissible open={filtersOpen}>
+                  <DrawerHeader>
+                    <DrawerTitle>
+                      <FormattedMessage id="filters.title" />
+                    </DrawerTitle>
+                    <DrawerSubtitle>
+                      <FormattedMessage id="filters.sub-title" />
+                    </DrawerSubtitle>
+                  </DrawerHeader>
+                  <DrawerContent style={{padding: '0px 16px'}}>
+                    <DateTimeRangePickerFormControl 
+                      width={'100%'} 
+                      renderAsButton={false} 
+                      onChange={(dateRange): void => {
+                        console.log('DateTimeRangePickerFormControl--->',dateRange.from, dateRange.to);
```

Comment:
console logs? 
if you need log use our logger

Reply:
It's still in development, basically logging no needed. 
When time filtering will be implemented console.log will vanish